### ISSUE TITLE
Get list of delivery places and fees without actually requesting an item

### ISSUE THREAD

**Issue by nichtich** [gbv/paia#70]
To better get the conditions required or available for a `request` or `renew`. Maybe add an explicit `/conditions` endpoint to get

- available condidtions for request
- available condidtions for renew

The answer may depend on user and document but it can be helpful nevertheless to get some default values, such as

* typical renewal fee (FeeCondition)
* typical list of places an item can be delivered to (StorageCondition)

The endpoint could also be accessible publicly for a "default user" with publicly known patron identifier:

**HTTP verb and URL**
    GET https://example.org/core/{uri_escaped_patron_identifier}/conditions

**scope**
  read_conditions

**request parameters**
* action (string): `request` or `renew`
* confirm (`0..1`) to select which types of conditions to check
* doc (`0..1`) to limit the conditions to a given document
* doc.item (`0..1`) URI of a particular item
* doc.edition (`0..1`) URI of a particular edition

**response fields**
* the response is [a condition](http://gbv.github.io/paia/paia.html#conditions-and-confirmations)

**Comment by ndege** [comment]
At first sight for me isn't obviously clear, what should be depict the parameter _confirm_ in this context?

**Comment by nichtich** [comment]
@ndege yes, the name is misleading. It could be used to select condition types, but the server could also just return all condition types so we can remove this parameter.
